Updating S3 API AWS CLI configuration (#7519)
ovh/docs: pages/storage_and_backup/object_storage/s3_getting_started_with_object_storage/guide.en-gb.md

aws s3 cp s3://<bucket_name> s3://<bucket_name_2> --recursive
```

> **Synchronizing buckets**

Reviewer: We usually prefer British English on documentation. 
```suggestion
**Synchronising buckets**
```

```bash

> aws s3 sync . s3://<bucket_name> # Synchronizing local directory to the S3 bucket

Reviewer: ```suggestion
aws s3 sync . s3://<bucket_name> # Synchronising local directory to the S3 bucket
```

> aws s3 sync s3://<bucket_name> . # Synchronizing S3 bucket to the local directory

Reviewer: ```suggestion
aws s3 sync s3://<bucket_name> . # Synchronising S3 bucket to the local directory
```

> aws s3 sync s3://<bucket_name> s3://<bucket_name_2> # Synchronizing a S3 bucket to another one

Reviewer: ```suggestion
aws s3 sync s3://<bucket_name> s3://<bucket_name_2> # Synchronising an S3 bucket to another one
```
